captionsProcessor.py

```diff
@@ -0,0 +1,25 @@
+from config import Config
+from transcriptLoader import getCombinedTranscripts
+from topicExtractor import retrieveTopics
+from utils import printAndLog
+
+
+def main():
+    config = Config()
+    config.setFromEnv()
+
+    config.videoToUse = "New Google Assignments in Canvas"
+
+    printAndLog(f"Retrieving Transcript for {config.videoToUse}")
+
+    transcriptToUse = getCombinedTranscripts(
+        config.videoToUse,
+        captionsFolder=config.captionsFolder,
+    )
+
+    printAndLog(f"Retrieving Topics for {config.videoToUse}")
+
+    topicsOverTime, topicModel = retrieveTopics(config.videoToUse, transcriptToUse, config=config)
```

Comment:
Have some Validation check around ` transcriptToUse ` value returned before passing it to `  retrieveTopics` This is essential and make the error tracking easier. As always code should be written to  take code of happy and unhappy path. The error case should give the user enough info to understand the issue and fix it.